### ISSUE TITLE
Tabindex values should equal 0

### ISSUE THREAD

**Issue by nickmarden** [GSA/search-gov#38]
We were alerted by a customer that our rails serp is failing their accessibility scanner, because we have `tabindex` greater than 0. Upon investigation, we've learned that `tabindex=0` is best practice, any other values are discouraged, and also that our rails serp doesn't have `tabindex=0` anywhere.

All verticals of rails SERPs should be checked and `tabindex` values updated to `0`. 

search_consumer SERPs can be ignored; see #34.

**Comment by nickmarden** [comment]
Anyone looking to take this on should communicate with the Search.gov team about which accessibility testers (e.g. WAVE, AXE, or others) should be considered authoritative for determining compliance of the current or modified SERPs.

**Comment by dawnpm** [comment]
False positive determined Nov 2018. See SRCH-163.
